### PR TITLE
add bfc-second-blog

### PR DESCRIPTION
**Pull request by lijiarui**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.
```
---
 
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [ ] Add `<!--more-->` section to just show the abstract in blog homepage.

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

File: docs/_posts/2019-07-26-bot-friday-second.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+1. 高原
+    1. 未来活动更多后，为活动打标签做区分。
+1. 刘晓倩
+    1. 倾向于圆桌会议形式举办活动。
+1. 俊良
+    1. 脑洞了一个“吐槽机器人”。
+
+### 3 Bot Friday 决议
+
+1. 巨头主席
```
Comment (huan) [inline docs/_posts/2019-07-26-bot-friday-second.md]: 巨头主席？平台主席？

File: docs/_posts/2019-07-26-bot-friday-second.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    1. 未来活动更多后，为活动打标签做区分。
+1. 刘晓倩
+    1. 倾向于圆桌会议形式举办活动。
+1. 俊良
+    1. 脑洞了一个“吐槽机器人”。
+
+### 3 Bot Friday 决议
+
+1. 巨头主席  
+Bot Friday 推选了首席巨头主席为会员提供巨头Chatbot 平台的行业指导，首席巨头主席为期1年，恭喜Bot Friday的首席巨头主席: 刘晓倩。
```
Comment (huan) [inline docs/_posts/2019-07-26-bot-friday-second.md]: “首席”巨头主席？“首位”巨头主席？

File: docs/_posts/2019-07-26-bot-friday-second.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    1. 李云军《Chatbot在团队任务管理中的应用》
+
+### 5 未来分享议题
+
+欢迎大家踊跃报名！
+
+1. 李明/俊良/Simon：Wechaty Mini Program Support
+1. 李明/俊良/Simon/云军：What will happen after Chatbot empowered by Mini Program?
+1. 俊良：如何用if else 和正则表达式写一个聊天机器人。
+1. 邝伟鹏：Chatbot 场景分享：秒印，驾校沟通的工具。
```
Comment (huan) [inline docs/_posts/2019-07-26-bot-friday-second.md]: 驾校 -> 家校

File: docs/_posts/2019-07-26-bot-friday-second.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+主场活动结束后，六个人前往上次After Party的管氏串吧继续补充能量，大家讨论了创业，社区，技术，招聘等一系列问题，并成功的消灭了3炮鲜啤！
+
+另外，After Party 上我们从Lua数组下标从1开始，聊到了本次活动的第一场是0还是1，最终投票了第一次活动应该叫做First！
+
+## 特别鸣谢
+
+1. 场地赞助方：腾讯公司
+1. 晚餐赞助方：Tencent Bot Platform产品团队
+1. 直播赞助方：毛豆教育公司
+1. 活动承办方：句子互动公司
```
Comment (huan) [inline docs/_posts/2019-07-26-bot-friday-second.md]: 承办方？主办方？

**Review by huan — APPROVED** [review]
Please confirm the reviews that I had left, and feel free to help yourself to merge this PR.

File: docs/_posts/2019-07-26-bot-friday-second.md (hunk 1 above)
Comment (lijiarui) [reply docs/_posts/2019-07-26-bot-friday-second.md]: 平台主席，我记错了。

**Comment by huan** [comment]
Is there any reason that we are using a `png` image file format instead of a standard `jpeg` format?

**Comment by lijiarui** [comment]
I compress the photos from big to small and find png is more clear compared with jpg in the same image size.

**Comment by huan** [comment]
Ok.

It seems that there was some trouble on the `github/pages` status update, I'd like to suggest you to add a new commit to this PR to trigger it again.

**Comment by huan** [comment]
It seems that the `github/pages` status will not update for a PR. So I remove the requirements for passing it before merging PRs.

**Pull request merged**